test/archethic_web/dashboard_aggregator_test.exs

```diff
@@ -0,0 +1,76 @@
+defmodule ArchethicWeb.DashboardAggregatorTest do
+  use ExUnit.Case
+
+  alias ArchethicWeb.DashboardAggregator
+  alias Archethic.PubSub
+
+  setup do
+    # the DashboardAggregator is already started and supervised
```

Comment:
This could be avoided by using the `enabled` attribute in test config. So the setup could be just to start the GenServer using `start_supervised`